src/static/css/content.less

```diff
@@ -533,19 +530,19 @@ figure {
 .demi-ex {
   p,
   li {
-    .u-webfont-demi;
+    font-weight: 600;
   }
 }
 .medium-ex {
   p,
   li {
-    .u-webfont-medium;
+    font-weight: 500;
   }
 }
 .regular-ex {
   p,
   li {
-    .u-webfont-regular;
+    font-weight: 400;
```

Comment:
Why 400 here and normal in other places?

Reply:
Sloppiness. I meant to use 400 everywhere. Will update.

Reply:
Fixed in 53ee82c